mcs/class/System.Private.CoreLib/System/Buffer.cs

```diff
@@ -17,7 +17,7 @@ public static void BlockCopy (Array src, int srcOffset, Array dest, int dstOffse
 			if (src == null)
 				throw new ArgumentNullException (nameof (src));
 			if (dest == null)
-				throw new ArgumentNullException (nameof (dest));
+				throw new ArgumentNullException ("dst");
```

Comment:
can't use nameof here, this method also throws "dest" in ArgumentException below (and there is a test that expects both "dst" and "dest" I guess it's a typo and should be fixed in the coreclr repo (and corefx test).

Reply:
That sounds like a typo, could you propose the change?